mds/migrations/0016_auto_20190320_1736.py

```diff
@@ -0,0 +1,127 @@
+# Generated by Django 2.1.3 on 2019-03-20 17:36
```

Comment:
I already took the `0016` position, should it matter.

Reply:
eheh I'll rebase no worry